plugin/path_checkout_sets.go

```diff
@@ -218,106 +218,109 @@ func (b *backend) operationReserveUpdate(ctx context.Context, req *logical.Reque
 
 		// For service accounts we won't be handling anymore, we need to remove their passwords and delete them
 		// from storage.
-		beingDeleted := strutil.Difference(reserve.ServiceAccountNames, newServiceAccountNames, true)
+		beingDeleted := strutil.Difference(set.ServiceAccountNames, newServiceAccountNames, true)
 		var deletionErrs error
 		for _, prevServiceAccountName := range beingDeleted {
-			if err := b.deleteReserveServiceAccount(ctx, req.Storage, prevServiceAccountName); err != nil {
+			if err := b.deleteSetServiceAccount(ctx, req.Storage, prevServiceAccountName); err != nil {
 				deletionErrs = multierror.Append(deletionErrs, err)
 			}
 		}
 		if deletionErrs != nil {
 			return nil, deletionErrs
 		}
-		reserve.ServiceAccountNames = newServiceAccountNames
+		set.ServiceAccountNames = newServiceAccountNames
 	}
 	if ttlSent {
-		reserve.TTL = ttl
+		set.TTL = ttl
+	}
+	if ttlSent {
+		set.TTL = ttl
```

Comment:
Looks like the merge double-created these two lines. I'll clean it up before merging.